compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/BaseVisitor.java

```diff
@@ -1037,4 +1037,6 @@ public void visit(BLangXMLNavigationAccess xmlNavigation) {
     @Override
     public void visit(BLangClassDefinition classDefinition) {
     }
+
+    public abstract void visit(BLangInvocation.BLangResourceAccessInvocation resourceAccessInvocation);
```

Comment:
@dulajdilshan, any reason why this method is defined as an abstract, instead of using 
`public void visit(BLangInvocation.BLangResourceAccessInvocation resourceAccessInvocation) {}`?